docs/tutorial/correlatedstacks.rst

```diff
@@ -118,4 +118,9 @@ downsampled over the video frames. This can be done using the function `Slice.do
 using timestamps obtained from the `CorrelatedStack`::
 
     # Determine the force trace averaged over frame 2...9.
-    file.force1x.downsampled_over(stack[2:10].frame_timestamp_ranges)
+    file.force1x.downsampled_over(stack[2:10].frame_timestamp_ranges())
+
+By default, this averages only over the exposure time of the images in the stack.
+If you wish to average over the full time range from the start of the scan to the next scan, pass the extra parameter `include_dead_time=True`::
+
+    file.force1x.downsampled_over(stack[2:10].frame_timestamp_ranges(include_dead_time=True))
```

Comment:
To me this flows much more intuitively!

Reply:
Yeah, I agree. Usually it's nicer when the default for something is `False`. Somehow it's more expected because you're asking for an action to be taken.